internal/controller/nnf_workflow_controller_helpers.go

```diff
@@ -694,6 +705,17 @@ func (r *NnfWorkflowReconciler) createNnfStorage(ctx context.Context, workflow *
 	return nnfStorage, nil
 }
 
+// trimMgsAddress takes an MGS address as given in the NnfStorageProfile ExternalMgs field
```

Comment:
Remove this comment block. You have it above..

Reply:
I forgot you guys get updates on my commits in this PR. It's not ready yet. I'll re-open after it's been tested